src/pspm_msg.txt

```diff
@@ -1,7 +1,7 @@
 $___________________________________________________________________________
 Welcome to PsPM - PsychoPhysiological Modelling
 (incorporating SCRalyze)
-Version 4.2.0 (17.06.2019)
+Version 4.2.1 (17.06.2019)
```

Comment:
Please also update the date. I think we can release today, so let us use today's date.